src/main/webapp/script.js

```diff
@@ -11,3 +11,18 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+// Make navbar stick to the top of the page when scrolling.
+window.onscroll = function () {
+  stickyNavbar();
+};
+
+function stickyNavbar() {
+  var navbar = document.getElementById("navbar");
+
+  if (window.pageYOffset >= 308) {
```

Comment:
I think these changes are already made in a different PR
https://github.com/googleinterns/step227-2020/pull/7/files#diff-dd236b8797a63209fda854e68262b51dR23